### DISCUSSION TITLE
Main menu disappeared and can't get it back

### DISCUSSION THREAD

**Discussion by vladatbint** [TabularEditor/TabularEditor3#1298] [🙋♀️TE3 Q&A] [answered]
### Description

After I enter my license key and load the model from PBI service Main menu was disappeared and I can't get it back from the Customization window.
In Customization window option Main menu is marked not checked and I can not change it.  

### Tabular Editor 3 Version

appear in 3.14 ( I installed 3.15 but was the same )

### Screenshots

![TE3 MM issue](https://github.com/TabularEditor/TabularEditor3/assets/169165993/40bfba16-8fc7-46ee-b93a-63a6277cfccb)

### Steps to Reproduce

I am not sure was it 100% like this but after loading a model from PBI service, I opened the window expression in order to see M code and then something happened with TE3 Srceen (for a short time the image flickered).
In first moment I didn't notice that the Main menu disappeared. After that I can not get it back.

### Windows Version

10

**Comment by otykier** [comment, marked as answer]
Hi @vladatbint 
Delete the `Layouts.json` file within the `%LocalAppData%\TabularEditor3` folder, and launch the application again.

**Reply by Mardurolla** [reply]
Win + r > paste %LocalAppData% >Press Enter > Select TabularEditor3 folder > delete layout.json
It worked for me but I now need to switch to another license key associated to a different email. How do I do that? Any one knows? Going to Help \About does not help.

**Reply by Mardurolla** [reply]
Ah, https://docs.tabulareditor.com/te3/getting-started.html#registry-details

**Comment by mlonsk** [comment]
Converting to a discussion Q&A as this is not an application bug, but rather a side effect of the customizability of Tabular Editor's UI.

**Comment by mlonsk** [comment]
This behaviour has been address in Tabular Editor 3.16.0

Menus and toolbars are now locked in place by default, preventing accidental repositioning. To unlock them, go to Tools > Customize... > Options and uncheck the Lock menus and toolbars option.
